shoppingcart.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,700;1,700&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="/shoppingcart.css" />
+  <head>
+   <style>
+     body {
+  background-image: url('https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2Fmobile%20green%20background.png?v=1604358440560');
+  background-size: 100%;
+font-family: 'Josefin Sans';
+  font-size:15px;
+  text-align:center;
+}
+
+#myBar{
+  background-color:#e26b2b;
+  width:0;
+  padding:10px;
+  border-radius:15px;
+    font-size:12px;
+  color:white;
+}
+
+#myProgress{
+  width:95%;
+  background-color:white;
+  border-radius:15px;
+}
+
+#myClick{
+  border:none;
+  border-radius:10px;
+}
+    </style>
+    <title>Finish You Pizza</title>
+    
+  </head>  
+  
+  <body>
+    <img class="logo" src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2F24403.png?v=1604357916769" alt="logo">
+    
+    <div id="mySidepanel" class="sidepanel">
+    <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">×</a>
+    <a href="1.html">Home</a>
+    <a href="new-pizza.html">New Pizzas</a>
+    <a href="about-blaze.html">Blaze Pizza</a>
+    <a href="about-us.html">About us</a>
+  </div>
+  
+  <button class="openbtn" onclick="openNav()">☰</button>  
+    
+    <h5 class='title'>Finish Your Pizza</h5>
+        <img id="myPizza" class="pizza" src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2Fblaze%20pizza%202.png?v=1604358058472"width="400" alt="pizza">
+     <div id = "myProgress">       
+    <div id = "myBar">0%</div>
```

Comment:
Be sure to run `npm run fix` on your repo **after** resolving the other issues here and merging everything in - it will automatically clean up your spacing for you.